src/components/App/MapRenderer/MapRenderer.module.scss

```diff
@@ -0,0 +1,4 @@
+.mapRenderer {
+  width: 50rem;
```

Comment:
How does this behave on smaller screen sizes? (Even if the answer is "badly", I think dealing with that should be on a separate branch so we can get this branch merged faster)

Reply:
it is